test/cluster_helper.go

```diff
@@ -381,6 +381,46 @@ func rollingRestartClusterByRemovingNamespaceDynamicallyTest(
 	return updateCluster(k8sClient, ctx, aeroCluster)
 }
 
+func validateServiceUpdate(k8sClient client.Client, ctx goctx.Context,
```

Comment:
I think service should be validated twice to check service ports are not overwritten :
1. Validate service when both tls and non-tls are configured in first step 
2. Validate service when final tls/non-tls is configured

Above scenarios are valid for both non-tls to tls migration and vice-versa